app/views/admin/merchants/index.html.erb

```diff
@@ -2,4 +2,9 @@
 
 <% @merchants.each do |merchant| %>
 <p><%= link_to merchant.name, admin_merchant_path(merchant) %></p>
+<% if merchant.status == "enabled" %>
+  <%= button_to "Disable #{merchant.name}", admin_merchant_path(merchant), method: :patch, params: { merchant: { status: "disabled"}} %>
+<% else merchant.status == "enabled" %>
```

Comment:
Should this be 'disabled'?

Reply:
Yes, good catch. I'll have to make my testing more robust too. Thanks Dani!

Reply:
@dani-wilson please see the update I just pushed. I removed the condition after `else` altogether since the attribute is an enum with only 2 values. It should logically read correctly now.. however I'm not totally sure why my testing suite was passing initially since I thought I covered that case in my rspec. Let me know if you see a specific scenario I should be unit testing for in the feature spec. Thanks!